contents/docs/error-tracking/assigning-issues.mdx

```diff
@@ -68,4 +67,21 @@ Issue assignment filters are evaluated in the order they are configured. They ca
 
 ### Alerting based on assignment 
 
-A common use case for automatic issue assignment is to alert assignees of new issues. Once the issues are automatically assigned, you can set up alerts to notify the assignee. See the [alerts](/docs/error-tracking/alerts) guide for more information.
\ No newline at end of file
+A common use case for automatic issue assignment is to alert assignees of new issues. Once the issues are automatically assigned, you can set up alerts to notify the assignee. See the [alerts](/docs/error-tracking/alerts) guide for more information.
+
+## Create external issues
+
+You can also create issues in external tracking systems like GitHub Issues, Linear, or GitLab. 
+
+First, set up an [integration](/docs/error-tracking/integrations) with your tracking system. Then, from an issue's details page, click **+ Create issue** under **External references**.
```

Comment:
```suggestion
First, set up an [integration](/docs/error-tracking/integrations) with your tracking system. Then, from an issue's details page, under **External references** click **+ Create issue**.
```

Readers should see External references before Create issue, helps them build the mental order of what to look for :)